Source/ACE.Database/CharacterDatabase.cs

```diff
@@ -49,7 +49,13 @@ private enum CharacterPreparedStatement
             CharacterPropertiesIntInsert,
             CharacterPropertiesBigIntInsert,
             CharacterPropertiesDoubleInsert,
-            CharacterPropertiesStringInsert
+            CharacterPropertiesStringInsert,
+
+            CharacterPropertiesBoolUpdate,
+            CharacterPropertiesIntUpdate,
+            CharacterPropertiesBigIntUpdate,
+            CharacterPropertiesDoubleUpdate,
+            CharacterPropertiesStringUpdate
         }
```

Comment:
Created some prepared statements that work with SQL UPDATE functions. I did not see any update functions listed, but I don't know if this is by design. This seems weird to me though, as the database guy inside of me is saying that we should use the correct functions to manipulate what's already available, instead of trying to over-write everything.